README.md

```diff
@@ -17,26 +17,46 @@ Finding a good issue: `↗`
+| **Purpose**      |                   Make it simple to manage payments, contributors and recipients                    |                               Raising donations and inform the public                               |                                                         Make it simple for recipients to manage payments and surveys                                                          |
+| **Instructions** |                                      [Readme](admin/README.md)                                      |                                     [Readme](website/README.md)                                     |                                              [Readme](recipients_app/README.md) / [Contributing](recipients_app/CONTRIBUTING.md)                                              |
+| **Localhost**    |               [localhost:3000](http://localhost:3000) / [4000](http://localhost:4000)               |                               [localhost:3001](http://localhost:3001)                               |                                                                                       –                                                                                       |
+| **Staging**      |              [staging-admin.socialincome.org](https://staging-admin.socialincome.org)               |                    [staging.socialincome.org](https://staging.socialincome.org/)                    |                                                                                  Testflight                                                                                   |
+| **Production**   |                      [admin.socialincome.org](https://admin.socialincome.org)                       |                           [socialincome.org](https://socialincome.org)\*                            | [iOS](https://apps.apple.com/ch/app/social-income/id6444860109?l=en-GB) / [Android](https://play.google.com/store/apps/details?id=org.socialincome.app&pcampaignid=web_share) |
+
+The website and admin tool use [cloud functions](functions/README.md).
+For frontend developers: you can also develop UI components with
 Tailwind CSS independent of the website ([Readme](ui/README.md) /
 [Contributing](ui/CONTRIBUTING.md)). The components are all collected in
 our [Storybook](https://socialincome-san.github.io/public/).
 
-\* The current website socialincome.org is still on a private repo. We
-are rebuilding the site with NextJS, Tailwind CSS and React on this
-public repo. You can visit the new website in the making on
+\* The website socialincome.org is still on a private repo. On this repo
+we currently rebuild the site with NextJS, Tailwind CSS and React. You
```

Comment:
In this repo, we are rebuilding the existing website with Next.js, DaisyUI, and Tailwind CSS.